src/test/java/com/training/assignments/Assignment3Test.java

```diff
@@ -0,0 +1,24 @@
+package com.training.assignments;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class Assignment3Test {
+    Assignment3 assignment3 = new Assignment3();
+
+    @Test
+    public void testGetIndex1() {
+        Assertions.assertEquals(0, assignment3.getIndex("Xioami", "X"));
```

Comment:
Xioami word used thrice can you please make separate local variable like String value = "Xioami";